bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/BlindHandler.java

```diff
@@ -0,0 +1,176 @@
+            case SHUTTER:
+                if (command.toFullString() == DOWN) {
+                    dominoswissHandler.continuousDown(id);
+                } else if (command.toFullString() == UP) {
+                    dominoswissHandler.continuousUp(id);
+                } else if (command.toFullString() == CHANNEL_STOP) {
+                    dominoswissHandler.stop(id);
+                } else {
+                    logger.debug("Blind got command but nothing executed: {}  and ChannelUID: {}",
+                            command.toFullString(), channelUID.getIdWithoutGroup());
+                }
+
+            case TILTDOWN:
+                if (command instanceof Number) {
+                    try {
+                        dominoswissHandler.tiltDown(id);
+                    } catch (InterruptedException e) {
+                        logger.error("EGate tiltDown error: {} ", e.toString());
+                    }
+                }
+                break;
+
+            case TILTUP:
+                if (command instanceof Number) {
+                    try {
+                        dominoswissHandler.tiltUp(id);
+                    } catch (InterruptedException e) {
+                        logger.error("EGate tiltUP error: {} ", e.toString());
+                    }
+                }
+                break;
+
+            case SHUTTERTILT:
+                if (command.toFullString() == UP) {
+                    dominoswissHandler.pulseUp(id);
+                } else if (command.toFullString() == DOWN) {
+                    dominoswissHandler.pulseDown(id);
+                } else if (command.toFullString() == CHANNEL_STOP) {
+                    dominoswissHandler.stop(id);
+                } else {
+                    logger.debug("Blind got command but nothing executed: {}  and ChannelUID: {}",
+                            command.toFullString(), channelUID.getIdWithoutGroup());
+                }
+
+            default:
+                break;
+        }
+    }
+
+    @SuppressWarnings("null")
+    @Override
+    public void initialize() {
+        this.id = getConfig().as(BlindConfig.class).id;
+        dominoswissHandler = (EGateHandler) getBridge().getHandler();
+        dominoswissHandler.registerBlind(this.id, getThing().getUID());
+        try {
+            ThingStatus bridgeStatus = getBridge().getStatus();
+            if (bridgeStatus == ThingStatus.ONLINE && getThing().getStatus() != ThingStatus.ONLINE) {
+                updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE);
+                dominoswissHandler = (EGateHandler) getBridge().getHandler();
+            } else if (bridgeStatus == ThingStatus.OFFLINE) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+            }
+        } catch (Exception e) {
+            logger.error("Could not update ThingStatus ", e);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE, e.toString());
```

Comment:
The log message could be removed, as the status update is already logged by the framework.

The state change originated by updateStatus() is logged to events.log. Including the status detail message.